src/main/handlers/Utilities/Prerequisites.ts

```diff
@@ -4,5 +4,11 @@ export const WindowsPrerequisites: AppModel[] = [
   getAppModel('wsl', 'Windows Subsystem for Linux (WSL)', 'wsl --status;', false),
   getAppModel('wslUbuntu', 'WSL Ubuntu Distribution', 'wsl --status;', false),
   getAppModel('dockerDesktop', 'Docker Desktop', 'docker version;', false),
-  getAppModel('dockerDesktopUbuntu', 'Docker Desktop WSL Ubuntu Integration', 'wsl docker version;', false)
+  getAppModel('dockerDesktopUbuntu', 'Docker Desktop WSL Ubuntu Integration', 'wsl docker version;', false),
+  getAppModel(
+    'ps1ExecutionPolicy',
+    'PowerShell Execution Policy',
+    'Get-ExecutionPolicy; $PSVersionTable.PSVersion',
```

Comment:
In my powershell 7 despite policy being unrestricted I am getting following error

![image](https://github.com/EtherealEngine/etherealengine-control-center/assets/10975502/043c08ea-ee21-4165-828c-13fe3d7c24b3)
![image](https://github.com/EtherealEngine/etherealengine-control-center/assets/10975502/f1e820f2-20d7-453f-955c-2c3c18cbffa2)